src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+    private final ColorSupplier colorSupplier;
+    private final Random random;
+
+    public Lottery(ColorSupplier colorSupplier, Random random) {
+        this.colorSupplier = colorSupplier;
+        this.random = random;
+    }
+
+    public Ball getRandomBall() {
+        Ball ball = new Ball();
+        ball.setNumber(random.nextInt(100));

```

Comment:
The magic number 100 should be a [constant](https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names)